src/loader/loader.cr

```diff
@@ -20,13 +20,13 @@ module BakedFileSystem
 
       result = [] of String
 
-      files = Dir.glob(File.join(root_path, "**", "*"))
+      files = Dir.glob(Path[root_path, "**", "*"].to_posix)
                  # Reject hidden entities and directories
                  .reject { |path| File.directory?(path) || !(path =~ /(\/\..+)/).nil? }
 
       files.each do |path|
         io << "bake_file BakedFileSystem::BakedFile.new(\n"
-        io << "  path:            " << path[root_path_length..-1].dump << ",\n"
+        io << "  path:            " << Path[path].relative_to(root_path).to_posix.to_s.dump << ",\n"
```

Comment:
I think `relative_to` isn't strictly necessary here. The string-based operation should be sufficient. `root_path` is a prefix of any file path discovered via glob on `#{root_path.to_posix}/**/*`, so we can simply chop that.
The only thing missing to make this entirely robust is `root_path = root_path.normalize` at the beginning of this method.
```suggestion
        io << "  path:            " << Path[path[root_path_length..]].to_posix.to_s.dump << ",\n"
```
I'm happy with keeping `relative_to`, though.

Reply:
Ah, right, yeah! I used `relative_to` as I was thought it might cover up any differences with the paths that are returned from `Dir.glob` and `root_path`, but as you mentioned and the fact they are both Windows paths the existing string-based logic should work fine?

I think as[ `root_path` is created via `File.resolve_path`](https://github.com/schovi/baked_file_system/blob/master/src/loader.cr#L3) we might not need to use `normalize`?